src/main/java/com/cms/claimmanagement/insuranceCompany/surveyor/controller/SurveyorController.java

```diff
@@ -2,33 +2,25 @@
 
 
 import com.cms.claimmanagement.insuranceCompany.surveyor.service.SurveyorService;
-import com.cms.claimmanagement.insuranceCompany.surveyor.surveyReport.SurveyReportDTO;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatusCode;
-import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.util.List;
 
-@RestController
+@RestController("/v1/surveyors")
 public class SurveyorController {
     @Autowired
     public SurveyorService surveyorService;
 
-    @GetMapping(path = "/surveyors/{estimatedLoss}")
-    public List<SurveyorResponseData> getSurveyorsByEstimateLimit(@PathVariable ("estimatedLoss") Integer estimatedLoss) {
+
+    @GetMapping(path = "/{estimatedLoss}")
```

Comment:
when we are seding a request for GET /v1/surveyors/1000, based on the rest convention, what I'm expecting is the Surveyor which the ID is 1000.

for this scenario, I think it would be better for the client to understand if you expect  a queryParam called estimatedLoss, like this: /v1/surveyors?estimatedLoss=1000

Reply:
that is a great insight for me. Thanks!